molecule_common/base/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM centos/systemd:latest
```

Comment:
Who is maintaining https://hub.docker.com/r/centos/systemd/? As a good practice, it would be good to depend on a tag other than `:latest` (e.g. `:7`), though it seems nothing other than `:latest` exist.